packages/mobx-state-tree/src/types/complex-types/model.ts

```diff
@@ -212,12 +209,10 @@ export interface IModelType<
         fn: (self: Instance<this>) => { actions?: A; views?: V; state?: VS }
     ): IModelType<PROPS, OTHERS & A & V & VS, CustomC, CustomS>
 
-    /** @deprecated See `types.snapshotProcessor` */
     preProcessSnapshot<NewC = ModelCreationType2<PROPS, CustomC>>(
         fn: (snapshot: NewC) => ModelCreationType2<PROPS, CustomC>
     ): IModelType<PROPS, OTHERS, NewC, CustomS>
 
-    /** @deprecated See `types.snapshotProcessor` */
```

Comment:
This line and the previous similar line on L215 are the only two relevant changes to this PR.